### PR TITLE
Add Ruby on Rails

### PR DESCRIPTION
**Pull request by Tyler887**
## Adding a language

- [x] The code displays "Hello World"
- [x] I have updated the readme to include the new language
- [x] I have incremented the language count in the readme
- [x] I have no association with the language
- [x] There are no copyright issues with this code (Contains copyrighted code, but I am complying with the license of the code.)

#### Link to programming language: 
https://rubyonrails.org

<sup>Attrib: guide https://guides.rubyonrails.org/getting_started.html under <a href="https://creativecommons.org/licenses/by-sa/4.0">Creative Commons Attribution-ShareAlike 4.0</a>,only modification is changing the task from article list to hello world.

### CODE REVIEW

**Review by MrBrain295 — CHANGES_REQUESTED** [review]
It should only be `Ruby on Rails`.

**Comment by Tyler887** [comment]
> It should only be `Ruby on Rails`.

@MrBrain295 Thanks, request @Richienb to review

File: readme.md [hunk 1]
```diff
@@ -727,6 +727,7 @@ Make sure to see [contributing.md](/contributing.md) for instructions on contrib
 * [RPG](r/RPG.rpgle)
 * [RTL2](r/RTL2.rtl2)
 * [Ruby](r/Ruby.rb)
+* [Rails](r/Rails.rb)<br /><sup>Attrib: guide https://guides.rubyonrails.org/getting_started.html under <a href="https://creativecommons.org/licenses/by-sa/4.0">Creative Commons Attribution-ShareAlike 4.0</a>,only modification is changing the task from article list to hello world.</sup>
```
Comment (MrBrain295) [inline readme.md:730]:
```suggestion
* [Rails](r/Rails.rb)
```

**Comment by 3in0** [comment]
Isn't Rails more of a framework based on Ruby?
https://stackoverflow.com/questions/1237372/what-is-the-difference-between-ruby-and-ruby-on-rails/26759107

**Comment by MrBrain295** [comment]
> Isn't Rails more of a framework based on Ruby? https://stackoverflow.com/questions/1237372/what-is-the-difference-between-ruby-and-ruby-on-rails/26759107

It is but be also accept things like libraries and frameworks (that's why we have things like React).

**Review by Richienb — CHANGES_REQUESTED** [review]
The file is still called "Ruby on Rails"

**Pull request merged**